### ISSUE TITLE
Quick Question

### ISSUE THREAD

**Issue by ModmenuFriday** [snipergaming888/snipergaming888.github.io#27]
Hey Sniper. How Come The Monika Mod Doesn't Work? and also why does cg5 take a long time to load my pc is fast but it takes a long time to load

**Comment by vackyton** [comment]
cg5 is bugged i think it takes too much ram so yea and in monika something is missing maybe the index.html?

**Comment by vackyton** [comment]
YOOOO I FOUND THE MONIKA BUG
so the link is https://snipergaming888.github.io/monika/
but the folder name i Monika
but the link is not capped monika
so go into index.html and do that :) @ModmenuFriday @snipergaming888

**Comment by ModmenuFriday** [comment]
its work

**Comment by ModmenuFriday** [comment]
it works here https://snipergaming888.github.io/Monika/

**Comment by vackyton** [comment]
yep :) i told u change the index.html to Monika instead of monika
